### PR TITLE
securityContext in prometheus-operator needs to be configurable

### PR DESCRIPTION
**Pull request by gangseok514**
the current securityContext is fixed. it needs to be configurable

Signed-off-by: gangseok.lee <gangseok.lee@samsung.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by paskal — APPROVED** [review]

File: stable/prometheus-operator/values.yaml [hunk 1]
```diff
@@ -530,6 +530,10 @@ prometheusOperator:
   #       - e2e-az1
   #       - e2e-az2
 
+  securityContext:
```
Comment (gianrubio) [inline stable/prometheus-operator/values.yaml:533]: please add this option on Readme.md

Comment (gangseok514) [reply stable/prometheus-operator/values.yaml:533]: @gianrubio Thanks, Done!

**Review by gianrubio — COMMENTED** [review]
/ok-to-test
/lgtm

**Comment by gangseok514** [comment]
@gianrubio Sorry, version conflict. plz /lgtm once more. Thank you!!

**Pull request merged**